index.js

```diff
@@ -1,9 +1,13 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
-module.exports = function(content) {
-	this.cacheable && this.cacheable();
-	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+module.exports = function(source) {
+  this.value = source;
+
+  var json = JSON.stringify(source)
+    .replace(/\u2028/g, '\\u2028')
+    .replace(/\u2029/g, '\\u2029');
+
+  return "module.exports = " + json;
```

Comment:
why not just `return "module.exports = " + '"' + source + '"'?

Reply:
Yeah I know. But why converting to json is better than wrapping the string with `"`?

Reply:
Really, it's a matter of choice.  I've been used to using json stringify for things like this where code gets evaluated.  But yeah, you are right, it is the same.